library/couchbase

```diff
@@ -1,11 +1,19 @@
 Maintainers: Couchbase Docker Team <docker@couchbase.com> (@cb-robot)
 GitRepo: https://github.com/couchbase/docker
 
-Tags: latest, enterprise, 7.0.0, enterprise-7.0.0
+Tags: latest, enterprise, 7.0.1, enterprise-7.0.1
+GitCommit: f412542829939319b5e722fa384c60b68e765bce
+Directory: enterprise/couchbase-server/7.0.1
+
+Tags: community, community-7.0.1
+GitCommit: f412542829939319b5e722fa384c60b68e765bce
+Directory: community/couchbase-server/7.0.1
+
+Tags: 7.0.0, enterprise-7.0.0
```

Comment:
Any particular reason to keep this version listed here?  Removing tags here will remove them from the "Supported" section on the Hub readme (and will prevent us from spending cycles rebuilding them on the official build servers), but the tags will still be available to users who want them.  (See https://github.com/docker-library/official-images#library-definition-files for more detail on this.)